aixplain/factories/file_factory.py

```diff
@@ -147,3 +148,15 @@ def create(
             license is not None if is_temp is False else True
         ), "File Asset Creation Error: To upload a non-temporary file, you need to specify the `license`."
         return cls.upload(local_path=local_path, tags=tags, license=license, is_temp=is_temp)
+
+    @classmethod
+    def download(cls, s3_path: Text, local_filename: Text = None) -> Text:
```

Comment:
Since regular users can not download files at the moment, I would remove this method.